### ISSUE TITLE
runscript not compatible with varnish 4.1

### ISSUE THREAD

**Issue by frisi** [collective/plone.recipe.varnish#35]
version 2.0a5 generates the following configration file for these settings:

settings

``` ini
[cache]
recipe =  plone.recipe.varnish:script
daemon = ${buildout:directory}/parts/varnish-build/sbin/varnishd
mode = foreground
bind = *:${ports:cache}
cache-size = 1G
user = ${users:cache}
varnish_version=4.1
telnet = ${hosts:cache-telnet}:${ports:cache-telnet}
configuration-file = ${buildout:directory}/production/cache.conf
```

start-script

``` bash
#!/bin/sh
exec .../parts/varnish-build/sbin/varnishd \
    -p user=nobody \
    -f ".../production/cache.conf" \
    -P ".../parts/cache/varnish.pid" \
    -a *:8151 \
    -T *:8152 \
    -s file,".../parts/cache/storage",1G \
    -F \
    "$@"
```

running `bin/cache` results in an error:

```
$ bin/cache 
Error:
Unknown parameter "user".
```

**Comment by frisi** [comment]
the -u parameter has been removed in varnish 4.1

for `varnish_version=4.1` `plone.recipe.varnish:script` should generate 
`-j unix,user=<username>`

instead of 
`-p user=<username>`


reading the docs (https://www.varnish-cache.org/docs/4.1/reference/varnishd.html#jail-options) it seems sufficient to simply start varnish with user root or varnish and omit the `-j` param (which is on by default) at all:

> With the unix jail technology activated, varnish will switch to an alternative user for subprocesses and change the effective uid of the master process whenever possible.
> The optional user argument specifies which alternative user to use. It defaults to varnish

**Comment by petschki** [comment]
this is fixed in master ... @mauritsvanrees can we have a release of that?

**Comment by mauritsvanrees** [comment]
Sure. I have released 2.0a8.

The first alpha release of version 2.0 was 2.5 years ago. I guess we should make a final release at some point. :-) Varnish 5.2 is out, so it would be good to support that. Pull request #38 started there, but should probably be revisited. But that could also be in a version 2.1 or 3. Maybe a discussion for a different issue.
